Django_Blog/blog/tests/test_models.py

```diff
@@ -0,0 +1,85 @@
+from django.test import TestCase
+from blog.models import Author,Blog,Comment
+
+class AuthorModelTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        Author.objects.create(name='Jennifer Dasal',about_author = 'Jennifer Dasal is the curator of modern and contemporary art at the North Carolina Museum of Art in Raleigh, North Carolina, and the host of the independent podcast ArtCurious, which she started in 2016 and which was named one of the best podcasts by O, The Oprah Magazine and PC Magazine. She holds an MA in art history from the University of Notre Dame and a BA in art')
+        
+    #testcase for check author name label 
+    def test_name_label(self):
+        author = Author.objects.get(id=1)
+        field_label = author._meta.get_field('name').verbose_name
+        self.assertEqual(field_label,'name')
+    
+    #testcase for check author biography label 
+    def test_about_author_label(self):
+        author = Author.objects.get(id=1)
+        field_label = author._meta.get_field('about_author').verbose_name
+        self.assertEqual(field_label,'about author')
+
+    #testcase for check author name maximum length
+    def test_name_max_length(self):
+        author = Author.objects.get(id=1)
+        max_length = author._meta.get_field('name').max_length
+        self.assertEqual(max_length,100)
+
+    #testcase for check author object name
+    def test_object_name(self):
+        author = Author.objects.get(id=1)
+        get_object_name = author.name
+        self.assertEqual(get_object_name,str(author))
+
+class BlogModelTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        author = Author.objects.create(name='Jennifer Dasal',about_author = 'Jennifer Dasal is the curator of modern and contemporary art at the North Carolina Museum of Art in Raleigh, North Carolina, and the host of the independent podcast ArtCurious, which she started in 2016 and which was named one of the best podcasts by O, The Oprah Magazine and PC Magazine. She holds an MA in art history from the University of Notre Dame and a BA in art')
+        Blog.objects.create(title='Test Blog', author=author ,blog_content='This is a test blog.')
```

Comment:
Please remove the space after `author=author`. You can use the black formatter to format python file.